### PR TITLE
Fix fio test for freebsd

### PR DESCRIPTION
**Pull request by dsrivastavv**

### CODE REVIEW

File: lisa/tools/lsblk.py [hunk 1]
```diff
@@ -100,6 +100,16 @@ def is_mounted(self) -> bool:
 
         return any(partition.mountpoint for partition in self.partitions)
 
+    @property
+    def is_resource_disk(self) -> bool:
```
Comment (squirrelsc) [inline lisa/tools/lsblk.py]: It's Azure specific, it should be in Azure feature.

File: lisa/tools/mkfs.py [hunk 2]
```diff
@@ -102,3 +120,60 @@ def _install(self) -> bool:
             package = "btrfsprogs"
         posix_os.install_packages(package)
         return self._check_exists()
+
+
+class BSDMkfs(Mkfs):
+    _PARTITION_ID_REGEX = re.compile(r"^(?P<disk>/dev/(da|nvd)\d+)p(?P<index>\d+)$")
```
Comment (squirrelsc) [inline lisa/tools/mkfs.py:128]: Add examples above.

File: lisa/tools/mkfs.py [hunk 3]
```diff
@@ -2,17 +2,31 @@
...
+    [
+        "xfs",
+        "ext2",
+        "ext3",
+        "ext4",
+        "btrfs",
+        "hugetlbfs",
+        "nfs",
+        "tracefs",
+        "ufs",
```
Comment (LiliDeng) [inline lisa/tools/mkfs.py:22]: I recall for freebsd, it also has zfs file system type?

Comment (dsrivastavv) [reply lisa/tools/mkfs.py:22]: Fio tests work with the most common disk format for the distro, which for bsd is `ufs`. There are other file systems as well, but they are currently not required so I skipped adding them.

File: lisa/tools/mkfs.py (hunk 2 above)
Comment (dsrivastavv) [reply lisa/tools/mkfs.py:128]: @squirrelsc I have fixed this comment. Can you please check?

**Comment by squirrelsc** [comment]
@LiliDeng LGTM

**Review by LiliDeng — APPROVED** [review]

**Pull request merged**